index.html

```diff
@@ -470,7 +470,7 @@ <h3>Prohibited States and Properties</h3>
 			</section>
 		<section id="mustContain">
 			<h3>Required Owned Elements</h3>
-			<p>Any <a>element</a> that will be <a>owned</a> by the element with this <a>role</a>. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>.</p>
+			<p>The <a>role</a> required for all <a>owned</a> children. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>. Authors MUST ensure all <a>owned</a> child nodes, and all elements referenced by <pref>aria-owns</pref> from the active element, have at least one of the roles specified in the <em>required owned element</em> for the role of the active element.</p>
```

Comment:
Using @jongund's words, with added markup:

```suggestion
			<p>The <a>role</a> required for all <a>owned</a> children of an element with a container role. For example, an element with the role <rref>list</rref> will own at least one child element with the role <rref>listitem</rref>. Authors MUST ensure at least one of the required roles is either a child element or referenced by <pref>aria-owns</pref> from the container element.</p>
```

Reply:
Do we need to define "container role"? Do we need this term? 

This update includes both "container role" and then ends with "container element". 

i'm counting 5 instances of "container role" in the spec now, where those instances could be revised to "container element with role..." or even dropping the word "container".